src/Taylor1.jl

```diff
@@ -837,6 +845,21 @@ function asin(a::Taylor1)
 end
 
 # Homogeneous coefficients for arcsin
+doc"""
+    asinHomogCoef(kcoef, ac, r_f,coeffs)
+
+Compute the `k-th` expansion coefficient of $c = \arcsin{a}$ given by
```

Comment:
I think you meant here $s=\arcsin{a}$, since below you use for the coefficients s_k. Also, I would change {a} by (a); it reads easier.